extensions/amp-analytics/0.1/linker-manager.js

```diff
@@ -187,11 +190,11 @@ export class LinkerManager {
    * @return {!Promise<string>} expanded template.
    */
   expandTemplateWithUrlParams_(template, expansionOptions) {
-    return variableServiceFor(this.ampdoc_.win)
-        .expandTemplate(template, expansionOptions)
+    const bindings = this.variableService_.getMacros();
+    return this.variableService_.expandTemplate(template, expansionOptions)
         .then(expanded => {
           const urlReplacements = Services.urlReplacementsForDoc(this.element_);
-          return urlReplacements.expandUrlAsync(expanded);
+          return urlReplacements.expandStringAsync(expanded, bindings);
```

Comment:
`expandUrlAsync`? to check protocol match

Reply:
reverted.